libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/extensions/MapboxNavigationEx.kt

```diff
@@ -0,0 +1,33 @@
+package com.mapbox.navigation.core.internal.extensions
```

Comment:
```suggestion
@file:JvmName("MapboxNavigationExtensions")

package com.mapbox.navigation.core.internal.extensions
```

To make it more Java user-friendly (remove the need of `Kt`).

Reply:
We used "ex" suffix for observer extensions. Used the same one here.

Reply:
There are some `public` extensions not using `Ex` https://github.com/mapbox/mapbox-navigation-android/blob/cf7a92f5c52f7230ce37996b25e8a77f1fd999b3/libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/extensions/ShieldEx.kt#L1 https://github.com/mapbox/mapbox-navigation-android/blob/cf7a92f5c52f7230ce37996b25e8a77f1fd999b3/libnavigation-base/src/main/java/com/mapbox/navigation/base/extensions/RouteOptionsEx.kt#L1 ¯\\\_(ツ)\_/¯

Reply:
OK, I went with Extensions after all.